hawk/routes/catcher/client.js

```diff
@@ -403,6 +439,27 @@ function handleMessage(message) {
     });
 }
 
+function getGetParams(url) {
```

Comment:
there is a built-in `url.parse()` method https://nodejs.org/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost